### ISSUE TITLE
Failing to set Minio Bucket Policy

### ISSUE THREAD

**Issue by rayblair06** [laravel/homestead#1260]
Fails to set Minio Bucket Policy, resulting in Vagrant stalling and responding with non-zero exit status.

## Versions
  - Vagrant: v2.2.4.
  - Provider: Virtualbox.
  - Homestead: v8.5.6.

## Host operating system
macOS High Sierra 10.13.6

## Homestead.yaml
```yaml
...
minio: true
...
- name: test-public
   policy: none
- name: test-private
   policy: none
...
```

## Expected behavior
Successfully create and apply bucket policy without, vagrant failing.

## Actual behavior
Fails to set Minio Bucket Policy, resulting in Vagrant stalling and responding with non-zero exit status

```
    homestead-7: Bucket created successfully `homestead/test-public`.
    homestead-7: Name:
    homestead-7:
    homestead-7:   mc policy - manage anonymous access to buckets and objects
    homestead-7: USAGE:
    homestead-7:   mc policy [FLAGS] PERMISSION TARGET
    homestead-7:   mc policy [FLAGS] FILE TARGET
    homestead-7:   mc policy [FLAGS] TARGET
    homestead-7:   mc policy list [FLAGS] TARGET
    homestead-7:
    homestead-7: FLAGS:
    homestead-7:   --recursive, -r
    homestead-7:        list recursively
    homestead-7:   --config-dir value, -C value  path to configuration folder (default: "/root/.mc")
    homestead-7:
    homestead-7:   --quiet, -q                   disable progress bar display
    homestead-7:   --no-color                    disable color theme
    homestead-7:   --json                        enable JSON formatted output
    homestead-7:   --debug                       enable debug output
    homestead-7:   --insecure                    disable SSL certificate verification
    homestead-7:   --help, -h
    homestead-7: show help
    homestead-7:
    homestead-7:
    homestead-7: PERMISSION:
    homestead-7:   Allowed policies are: [none, download, upload, public].
    homestead-7:
    homestead-7: FILE:
    homestead-7:   A valid S3 policy JSON filepath.
    homestead-7:
    homestead-7: EXAMPLES:
    homestead-7:    1. Set bucket to "download" on Amazon S3 cloud storage.
    homestead-7:       $ mc policy set download s3/burningman2011
    homestead-7:
    homestead-7:    2. Set bucket to "public" on Amazon S3 cloud storage.
    homestead-7:       $ mc policy set public s3/shared
    homestead-7:
    homestead-7:    3. Set bucket to "upload" on Amazon S3 cloud storage.
    homestead-7:       $ mc policy set upload s3/incoming
    homestead-7:
    homestead-7:    4. Set policy to "public" for bucket with prefix on Amazon S3 cloud storage.
    homestead-7:       $ mc policy set public s3/public-commons/images
    homestead-7:
    homestead-7:    5. Set a custom prefix based bucket policy on Amazon S3 cloud storage using a JSON file.
    homestead-7:       $ mc policy set-json /path/to/policy.json s3/public-commons/images
    homestead-7:
    homestead-7:    6. Get bucket permissions.
    homestead-7:       $ mc policy get s3/shared
    homestead-7:
    homestead-7:    7. Get bucket permissions in JSON format.
    homestead-7:       $ mc policy get-json s3/shared
    homestead-7:
    homestead-7:    8. List policies set to a specified bucket.
    homestead-7:       $ mc policy list s3/shared
    homestead-7:
    homestead-7:    9. List public object URLs recursively.
    homestead-7:       $ mc policy --recursive links s3/shared/
    The SSH command responded with a non-zero exit status. Vagrant
    assumes that this means the command failed. The output for this command
    should be in the log above. Please read the output to determine what
    went wrong.
```

**Comment by rayblair06** [comment]
## Possible Fix
```
mc policy set $2 homestead/$1
```

Instead of

```
mc policy $2 homestead/$1
```

https://github.com/laravel/homestead/pull/1261

**Comment by svpernova09** [comment]
Fixed via #1261
